benchmarks/benchmark_run.sh

```diff
@@ -0,0 +1,168 @@
+#! /bin/bash
+set -e
+
+OMP_NUM_THREADS=24 # need to be modified based on cpus_per_worker
+VALUE_INF=2000
+choice=${1}
+
+get_peak_throughpt(){
+    bs=${1}
+    num_prompts=${2}
+    log_path=${3}
+    if [ -f $log_path ]; then
+        rm $log_path
+    fi
+    for vllm_bs in ${bs[*]};
+    do
+        echo "RUN VLLM"
+        echo "RUN bs ${vllm_bs}"
+        echo "bs: ${vllm_bs}" >> $log_path
+        # server:
+        OMP_NUM_THREADS=$OMP_NUM_THREADS numactl -N 0 -m 0 -C 0-$(($OMP_NUM_THREADS-1)) llm_on_ray-serve --config_file llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml --simple --max_concurrent_queries $VALUE_INF --vllm_max_num_seqs $vllm_bs
```

Comment:
could we define numactl -N 0 -m 0 -C 0-$(($OMP_NUM_THREADS-1)) to some variable so that don't need to duplicate this line again and again.